### PR TITLE
Added the support for MuKeyboard

### PR DESCRIPTION
**Pull request by MrMoDDoM**
Hi! 
I added the support for the MuKeyboard library: this should works without any problems..!

I basically copied the file Arduino.js and modified every "dalay(" with "wait(", and changed the #import from "Keyboard.h" to "MuKeyboard.h"

Cheers

-MrMoDDoM

### CODE REVIEW

File: modules/MuKeyboard.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+      if(µ.keyMap[argList[0]] !== undefined) {
+        return '  typeKey(\'' + µ.keyMap[argList[0]] + '\');\n\n';
+      } else {
+        µ.throwError("Unknown command or key: '" + argList[0] + "'");
+        return;
+      }
+    },
+
+    STRING: function(argList, µ) {
```
Comment (Plazmaz) [inline modules/MuKeyboard.js]: Please remove the  µ(mu) character from this. It's an anti-pattern to use non-alphanumeric variable names. Additionally if you use it in a string, please use the proper escape code for it.

File: modules/MuKeyboard.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+      // Replace all '"' by '\"' and all '\' by '\\'
+      textString = textString.split('\\').join('\\\\').split('"').join('\\"');
+
+      if(textString !== '') {
+        return '  Keyboard.print(F("' + textString + '"));\n\n';
+      }
+    },
+
+    DELAY: function(argList, µ) {
```
Comment (Plazmaz) [inline modules/MuKeyboard.js]: Sam here.

File: modules/MuKeyboard.js (hunk 1 above)
Comment (Plazmaz) [reply modules/MuKeyboard.js]: Just realized this is in the core codebase, sorry. CC @Nurrl

**Comment by Nurrl** [comment]
@MrMoDDoM @Plazmaz That sounds good, but I'll need to merge `LocaleKeyboard.js` into Duckuino for 'per module locale generation'.

**Pull request merged**